apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -981,8 +982,8 @@ const Ticket = new GQLListSchema('Ticket', {
             },
             {
                 type: 'BTreeIndex',
-                fields: ['assignee', 'executor', 'organization'],
-                name: 'ticket_assignee_exec_org',
+                fields: ['organization', 'assignee', 'executor', 'deletedAt'],
+                name: 'ticket_org_assignee_exec_del',
```

Comment:
please specify all names fully. It's not useful  to cut it there

Reply:
the max amount of characters for index naming is 32, but i made it more understandable